app/src/main/java/org/mozilla/fenix/settings/account/AccountSettingsFragment.kt

```diff
@@ -166,12 +165,7 @@ class AccountSettingsFragment : PreferenceFragmentCompat() {
         val historyNameKey = getPreferenceKey(R.string.pref_key_sync_history)
         findPreference<CheckBoxPreference>(historyNameKey)?.apply {
             setOnPreferenceChangeListener { _, newValue ->
-                requireComponents.analytics.metrics.track(Event.PreferenceToggled(
```

Comment:
We are already tracking these events in `SettingsFragment` so we were getting 2 pings here every time someone changed their setting.